crates/storage_models/src/schema.rs

```diff
@@ -198,6 +198,9 @@ diesel::table! {
         amount_captured -> Nullable<Int8>,
         connector -> Varchar,
         connector_mandate_id -> Nullable<Varchar>,
+        start_date -> Nullable<Timestamp>,
+        end_date -> Nullable<Timestamp>,
+        meta_data -> Nullable<Jsonb>,
```

Comment:
can we follow the same convention as all the other tables, please rename it to `metadata`